src/utils/bitfield.ts

```diff
@@ -6,34 +6,36 @@ export type BitFieldResolvable =
   | string
   | string[]
   | BitField[]
+  | bigint
+  | Array<bigint>
```

Comment:
`bigint[]` seems better

Reply:
![image](https://user-images.githubusercontent.com/43033058/108715861-68697d80-7541-11eb-89d1-eda2a25e2502.png)

Maybe not yet.